static/katex.less

```diff
@@ -232,8 +232,25 @@
         & + .mop.mtight { margin-left: @thinspace; }
     }
 
+    .vlist-t {
+        display: inline-table;
+        table-layout: fixed;
+    }
+
+    .vlist-r {
+        display: table-row;
+    }
+
+    .vlist-s {
+        display: table-cell;
+        vertical-align: bottom;
+        font-size: 0.05em;
```

Comment:
Is the 0.05em b/c we want the impact of the font size on the base line of elements with this class to be negligible?